### PR TITLE
Use colorPrimary for the "edit" schedule icon

### PR DESCRIPTION
**Pull request by nikclayton**
The "edit" icon when showing a scheduled status' time was grey, so it's not obvious that this section is clickable.

Use colorPrimary, so it looks more like a button.

### CODE REVIEW

File: app/src/main/res/layout/view_compose_schedule.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         android:paddingBottom="16dp"
         android:textColor="?android:textColorTertiary"
         android:textSize="?attr/status_text_medium"
-        app:drawableTint="?android:attr/textColorTertiary"
+        app:drawableTint="?attr/colorPrimary"
```
Comment (Lakoja) [inline app/src/main/res/layout/view_compose_schedule.xml:26]: This is a textview that is clickable?

**Comment by Lakoja** [comment]
What do I need to do to see this?

File: app/src/main/res/layout/view_compose_schedule.xml (hunk 1 above)
Comment (nikclayton) [reply app/src/main/res/layout/view_compose_schedule.xml:26]: Yes, clicking anywhere on the textview allows the user to reschedule the post.

Arguably all the text should be colorPrimary to indicate that the whole thing is clickable, but then there's quite a lot of blue in a small area and I found it distracting.

**Comment by Lakoja** [comment]
>     3. Click the "clock" icon again to see the scheduled date/time

Haven't seen that, yet. Looks a bit dodgy. (Texts not aligned, for example)

But this is an improvement then. And coloring only the pen is probably correct also.

**Review by Lakoja — APPROVED** [review]

**Pull request merged**